auth.go

```diff
@@ -23,6 +24,20 @@ type Authorization struct {
 	Permissions []Permission `json:"permissions"`
 }
 
+// Valid ensures that the authorization is valid.
+func (a *Authorization) Valid() error {
+	for _, p := range a.Permissions {
+		if p.OrgID != a.OrgID {
```

Comment:
Zero value also works here.  Is there already a check for p.OrgID being valid?

Reply:
There are other checks, but I'll add one here to ensure that the provided orgID is valid.